pkg/annotations/annotations.go

```diff
@@ -6,4 +6,8 @@ const (
 
 	// MultipleTemplatesSupportedAnnotation is an annotation that indicates whether multiple templates of the same kind are supported by the template's remediator
 	MultipleTemplatesSupportedAnnotation = "remediation.medik8s.io/multiple-templates-support"
+
+	// NodeNameAnnotation is an annotation that will be placed on the CRs of remediatiors who support multiple templates of the same remediator.
```

Comment:
Typo: remediatiors -> remediatiors

Reply:
>Typo: remediatiors -> remediatiors

Thanks, although your replace suggest the same text in both occurrences.
Did you mean remediatiors -> remediators ?

Reply:
:D a typo in a comment about typo. 

> Did you mean remediatiors -> remediators ?

yes :)